### PR TITLE
chore(typescript): Add better TypeScript definitions for Angular

### PR DESCRIPTION
**Pull request by cmckni3**

### CODE REVIEW

File: packages/okta-angular/src/okta/services/okta.service.ts [hunk 1]
```diff
@@ -63,14 +63,14 @@ export class OktaAuthService {
...
       return this.oktaAuth;
     }
 
     /**
      * Checks if there is a current accessToken in the TokenManager.
      */
-    isAuthenticated() {
+    isAuthenticated(): boolean {
       return !!this.oktaAuth.tokenManager.get('accessToken');
     }
```
Comment (cmckni3) [inline packages/okta-angular/src/okta/services/okta.service.ts]: Not sure what the type definitions of `getAccessToken` and `getIdToken` should be. Does `@okta/okta-auth-js` contain a structure for it?

Comment (cmckni3) [reply packages/okta-angular/src/okta/services/okta.service.ts]: Looks like #115 corrects it.

File: packages/okta-angular/src/okta/models/auth-required-function.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { Router } from '@angular/router';
```
[the remark this answers is not in the document]
Comment (cmckni3) [reply packages/okta-angular/src/okta/models/auth-required-function.ts]: Done

**Comment by cmckni3** [comment]
Got it. Had to fix the type of `onAuthRequired` in the test harness `app.module.ts`.

`npm start` looks good on my end. `npm test` times out.

File: packages/okta-react/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@okta/okta-react",
-  "version": "0.0.11",
+  "version": "0.0.12",
```
[the remark this answers is not in the document]
Comment (cmckni3) [reply packages/okta-react/package.json]: Fixed. I had branched from master so there was an extra commit.

**Comment by cmckni3** [comment]
sweet. Thanks, @jmelberg-okta!

**Comment by jmelberg-okta** [comment]
Closing in favor of #155 (containing these changes by @cmckni3)

**Pull request closed without merging**